apartment/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import include, path
+from rest_framework.routers import DefaultRouter
+
+from apartment.api.views import ProjectAPIView
+
+router = DefaultRouter()
+
+urlpatterns = [
+    path("projects/", ProjectAPIView.as_view(), name="project-list"),
```

Comment:
While this is OK for now, I'm wondering if this endpoint is going to start including some sensitive data at some point. The previous PR that you made put the endpoint behind /sales/, maybe it would be a good idea to do the same here just to keep things consistent? Or will this endpoint be used for more than the sales tool?

Reply:
Good question. The apartments can be shared as there isn't anything sensitive data. It is very general information. But for now, this is only used for the Sales tool.

Reply:
I'll change this that it is under the `sales`. We can improve that if that is necessary open for wider.

Reply:
This is now changed.